docs/UserGuide.md

```diff
@@ -67,16 +67,18 @@ The `Add Transaction` command allows you to add either an income or expense entr
 <a id="param_details"></a>
 **Parameter Details**:
 - `DESCRIPTION`: A brief label describing the transaction (e.g., "Lunch" or "Freelance Work").
+  - Note: Shouldn't be blank. And DO NOT USE `¦¦` and `/` in `DESCRIPTION` cause it serve as the separator token in storage file and the prefix for command arguments.
```

Comment:
Perhaps you might want to review the grammar here (and other edited parts as well)?